simulariumio/cellpack/cellpack_converter.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import os
+
+import numpy as np
+from scipy.io.arff.arffread import MetaData
```

Comment:
Is this right? Shouldn't `MetaData` be imported from `data_objects`?

Reply:
yea i'm surprised this didn't give me an error of some kind. It must have been an auto import that I didn't notice

Reply:
Ah I hate when that happens